lib/whats/client.rb

```diff
@@ -4,21 +4,22 @@
 
 module Whats
   class Client
-    def initialize(login = Whats::Actions::Login.new)
+    def initialize(token = nil, token_type = :bearer)
```

Comment:
You defined default `token_type` as `:bearer` here, but you're always passing `:basic`on `Actions::Login`, why you don't use `:basic` as default here?

_lib/whats/actions/login.rb_
```ruby
      def initialize
        @user = Whats.configuration.user
        @password = Whats.configuration.password
        @client = Whats::Client.new(encoded_auth, :basic)
      end
```

Reply:
@OtavioHenrique I'm keeping the client with previous defaults, you can see in the line 18, the token type is Bearer, so basically the changes here is just to bring the Login client options, the `Whats::Client.new` used in [api/whats/api.rb](https://github.com/getninjas/whatsapp/blob/master/lib/whats/api.rb#L51) was not touched. The another reason to keep Bearer as default token type is because the default Whatsapp communication use Bearer.